src/components/MoneyRaisedDisplay.js

```diff
@@ -23,15 +23,22 @@ class MoneyRaisedDisplay extends React.Component {
       textVariant,
       longCopy,
       excludeText = false,
+      moneyRaised,
     } = this.props
+    if (moneyRaised === null) {
```

Comment:
Q: is this breaking for other places that use this component?

Reply:
I checked the homepage (eg / path) and it worked since it's null vs. undefined. I can double check though.

Reply:
OK, that works.